README.md

```diff
@@ -1,29 +1,123 @@
 # Quarkus Test Suite
 
+The aim of this test suite is to validate Quarkus framework features along the time. Under the hood we are using [Quarkus-test-framework](https://github.com/quarkus-qe/quarkus-test-framework) that we designed for this proposal. 
```

Comment:
Not the main aim of this test suite. Can be omitted IMO. I guess you wanted to reference another chapter https://github.com/quarkus-qe/quarkus-test-suite#test-framework which mentioned in https://github.com/quarkus-qe/quarkus-test-suite/issues/118